submissions/fernando/index.html

```diff
@@ -48,38 +51,68 @@
             margin: 10px;
             align-self: flex-end;
         }
+        .aiAnswer {
```

Comment:
This works, but by convention, we would write:
```suggestion
        .ai-answer {
```